_posts/2019-12-09-reinvent-2019-recap.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "re:Invent 2019 Recap"
+date: 2019-12-09
+tags: aws reinvent recap
+author: Nathan Glover
+image: /img/blog/reinvent-2019-recap/reinvent-opener.png
+---
+
+## Introduction
+
+{:refdef: style="text-align: center;"}
+<img src="/img/blog/reinvent-2019-recap/reinvent-opener.png" width="700">
+{: refdef}
+
+re:Invent 2019 is the largest Amazon run event that occurs yearly in Las Vegas. Since 2011 it has been progressively scaling up in size and this year I was lucky enough to be able to attend for the first time.
+
+I'm sure many readers have filled themselves in on the announcements remotely; however as someone who; in 2018 tried to digest what releases were *important*; I've decied to put together this post outlining the things that stood out to me.
+
+### Fargate
+
+{:refdef: style="text-align: center;"}
+<img src="/img/blog/reinvent-2019-recap/reinvent-fargate.png" width="700">
+{: refdef}
+
+When fargate launched at re:Invent 2017 there was a lot of excitement from anyone who had been running container workloads. Fargate allowed developers to freely scale their workloads **UP** and **OUT** with very minimal configuration and only pay for the time those containers were actually running.
+
+A few years later and developers have been looking at Fargate for more features, and we were lucky enough this year to see a few of these launch!
+
+#### [Fargate for EKS](https://aws.amazon.com/blogs/aws/amazon-eks-on-aws-fargate-now-generally-available/)
+
+{:refdef: style="text-align: center;"}
+<img src="/img/blog/reinvent-2019-recap/reinvent-fargate-eks.png" width="700">
+{: refdef}
+
+Kubernetes has grown in popularity a lot over the last couple years, however there's a lot of overhead required to manage a Kuberenetes cluster; and this overhead has become just another thing developers need to learn in order to deploy their applications.
```

Comment:
"Kuberenetes" -> Kubernetes

Reply:
Don't think you should join sentences with a `;` and `and`